### ISSUE TITLE
question about the performance of the pretrained model

### ISSUE THREAD

**Issue by YoungZiyu** [sshaoshuai/PointRCNN#107]
Hi @sshaoshuai ！
Thanks for your great work！I find something interesting when i inference your pre-trained model.
when I set batch size == 1, I got a same performance as yours.
![Screenshot from 2019-09-18 13-39-24](https://user-images.githubusercontent.com/37181946/65114775-c2dfdf80-da19-11e9-8ff0-63c99aaa713f.png)
but when I set batch size == 2, I got 
![Screenshot from 2019-09-18 13-36-59](https://user-images.githubusercontent.com/37181946/65114460-6f6d9180-da19-11e9-9b91-4977b65f52ae.png)
and when I set batch size == 4, I got
![Screenshot from 2019-09-18 13-38-44](https://user-images.githubusercontent.com/37181946/65114701-b22f6980-da19-11e9-95bd-08030ff426b0.png)

The result should be consistent after 'model.eval()' is set and same random seed is used, and I don't know why it's changeable according to the batch size.

**Comment by alar0330** [comment]
> The result should be consistent after 'model.eval()' is set and same random seed is used

I am not sure about this statement, the seed might be the same, but the fact that you sample differently your batches could play a role, since e.g. `random_sampling` is done on the fly one batch at a time.

**Comment by 1992Zz** [comment]
Hello, the download link of the pre-trained model has been unable to open. Can you share yours with the trained model?
